### DISCUSSION TITLE
How to make checkbox with image.

### DISCUSSION THREAD

**Discussion by jigujigu** [filamentphp/filament#8559] [Help]
Hi All,
How to make checkbox with image, like this one in filamentphp v3.

![image](https://github.com/filamentphp/filament/assets/6214964/0d2ad71a-22eb-4ad3-ae55-0c4001ea9880)
Thank You

**Comment by leandrocfe** [comment]
You can create a [custom field](https://filamentphp.com/docs/3.x/forms/fields/custom#custom-field-classes).
There is a [similar field](https://filamentphp.com/plugins/ibrahim-bougaoua-radio-btn-image) using radio element.
